generator/pom.xml

```diff
@@ -128,6 +128,12 @@
             <version>0.11</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Should we mention somewhere that we use now use this framework for testing (@cakehurstryan, what are your thoughts?).

Do we need to make reference to this in the project as far as open sourcing/attribution?

Reply:
I think members of the open source community will probably be aware of mockito for unit testing, so by having it in the dependencies people would probably know what it's for.

Reply:
I understand that this can now be resolved